src/analyze/analyze.c

```diff
@@ -294,7 +296,12 @@ static int acquire_boot_times(sd_bus *bus, struct boot_times **bt) {
                 return -EINPROGRESS;
         }
 
-        if (arg_user) {
+        if (arg_scope == UNIT_FILE_SYSTEM) {
+                if (times.initrd_time)
```

Comment:
coding style: `times.initrd_time > 0`

Reply:
That was existing code.

Reply:
Updated to fix this and similar style issues.

Reply:
Thank you and sorry for requesting such a cleanup.